swagger/v2/swagger.json

```diff
@@ -2089,6 +2089,13 @@
             "unknown"
           ]
         },
+        "hypertension": {
```

Comment:
`v2` should not have this field in the API. This is just an artifact of how swagger functions, not how you've implemented it.

It needs to be elided from the v2 [payload](https://github.com/simpledotorg/simple-server/blob/master/spec/factories/medical_histories.rb#L57) in the factory which is used in this [spec](https://github.com/simpledotorg/simple-server/blob/master/spec/api/v2/medical_histories_spec.rb#L19) (that which auto-generates the swagger schema)

Reply:
Interesting. I will go exclude this field from that payload for now.

Are we concerned that updates to the current API have these trickledown effects to previous versions like v2? I'd assume that v2 should remain fixed in the face of changes to the current API.

Reply:
I _think_ the Swagger definitions are being updated because `Api::V2::Models` inherit from `Api::Current::Models`. I think the factory usage you linked to can remain unchanged, but I'll need to delete some schema keys in the V2 models [like we do for appointments](https://github.com/simpledotorg/simple-server/blob/master/app/schema/api/v2/models.rb#L30-L35).

Reply:
Ah, yep, you figured it out. That's the correct way. Apologies I missed that part in my comment.